### PR TITLE
Add ale_go_govet_options variable

### LINKED ISSUE #1817: [RFC] Please add a variable to customize 'go vet' arguments
It seems `gofmt` has a variable `g:ale_go_gofmt_options`, but `go vet` doesn't.
I appreciate if such variable exists.

For example, I want to supply `-composites=false` to `go vet` argument.

```
let g:ale_go_govet_options = '-composites=false'
```

### PR DESCRIPTION
**Pull request by tyru**
Fix #1817

### CODE REVIEW

**Comment by tyru** [comment]
I tried to add #1817 variable.
Sorry, I don't have docker environment (because WSL debian now) so couldn't run `./run-tests` script.
And also don't have vador environment.
But I think it's OK because it seems CI checks them.

**Comment by tyru** [comment]
Now all checks are passed, rebased it.

**Comment by tyru** [comment]
oops, PR title and commit message is wrong... fixed it. s/gofmt/govet/

File: ale_linters/go/govet.vim [hunk 1]
```diff
@@ -4,8 +4,12 @@
 " Author: John Eikenberry <jae@zhar.net>
 " Description: updated to work with go1.10
 
+call ale#Set('go_govet_options', '')
+
 function! ale_linters#go#govet#GetCommand(buffer) abort
+    let l:options = ale#Var(a:buffer, 'go_govet_options')
     return ale#path#BufferCdString(a:buffer) . ' go vet .'
+    \   . (!empty(l:options) ? ' ' . l:options : '')
```
Comment (w0rp) [inline ale_linters/go/govet.vim]: You can use `ale#Pad` for the same thing too, but this is fine.

**Comment by w0rp** [comment]
Cheers! :beers:

**Comment by tyru** [comment]
Thanks! :tada:

File: ale_linters/go/govet.vim (hunk 1 above)
Comment (tyru) [reply ale_linters/go/govet.vim]: I copied it from ale_linters/go/gobuild.vim .
I think other linters may have it.

**Pull request merged**